extensions/industry/healthcare/fhir/fieldgroups/patient.schema.json

```diff
@@ -0,0 +1,241 @@
+                        "description":"Time period when the contact point was/is in use.",
+                        "$ref":"https://ns.adobe.com/xdm/industry/healthcare/fhir/datatypes/period"
+                     }
+                  }
+               }
+            },
+            "xdm:communication":{
+               "title":"Communication",
+               "description":"A language which may be used to communicate with the person about his or her health.",
+               "type":"array",
+               "items":{
+                  "type":"object",
+                  "additionalProperties":false,
+                  "properties":{
+                     "xdm:language":{
+                        "title":"Language",
+                        "description":"The language which can be used to communicate with the person about his or her health.",
+                        "$ref":"https://ns.adobe.com/xdm/industry/healthcare/fhir/datatypes/codeableconcept"
+                     },
+                     "xdm:preferred":{
+                        "title":"Is Preferred Language",
+                        "type":"boolean",
+                        "description":"Language preference indicator"
+                     }
+                  }
+               }
+            },
+            "xdm:generalPractioner":{
+               "title":"General Practioner",
+               "description":"Patient's nominated primary care provider.",
+               "type":"array",
+               "items":{
+                  "$ref":"https://ns.adobe.com/xdm/industry/healthcare/fhir/datatypes/reference"
+               }
+            },
+            "xdm:managingOrganization":{
+               "title":"Managing Organization",
+               "description":"Organization that is the custodian of the patient record.",
+               "$ref":"https://ns.adobe.com/xdm/industry/healthcare/fhir/datatypes/reference"
+            },
+            "xdm:link":{
+               "title":"Patient Link Details",
+               "description":"Link to a Patient or RelatedPerson resource that concerns the same actual individual.",
+               "type":"array",
+               "items":{
+                  "type":"object",
+                  "additionalProperties":false,
+                  "properties":{
+                     "xdm:other":{
+                        "title":"Other",
+                        "description":"Link to a Patient or RelatedPerson resource that concerns the same actual individual.",
+                        "$ref":"https://ns.adobe.com/xdm/industry/healthcare/fhir/datatypes/reference"
+                     },
+                     "xdm:type":{
+                        "title":"Type",
+                        "description":"The type of link between this patient resource and another patient resource.",
+                        "type":"string"
+                     }
+                  }
+               }
+            }
+         }
+      }
+   },
+   "allOf":[
```

Comment:
You will also need to include all datatype references used in this file into the "allOf" section, for example: 
"allOf": {
    {
        "$ref":"https://ns.adobe.com/xdm/industry/healthcare/fhir/datatypes/identifier"
    },
    {
         "$ref":"https://ns.adobe.com/xdm/industry/healthcare/fhir/datatypes/humanname"
    },
... and so on